specification/transport_layer/transport_layer.tex

```diff
@@ -379,7 +386,11 @@ \subsection{Redundant interface support}
 utilization rates of the redundant interfaces and different phasing of their traffic;
 however, that is not an issue for UAVCAN.
 If perfectly simultaneous frame submission is not possible, interfaces with lower numerical index
-should be handled in the first order.
+should be handled in the first order\footnote{It should be kept in mind that redundant interfaces
+are used for increased fault tolerance, not for load sharing reasons.
```

Comment:
I think this first sentence is superfluous to the footnote and could be omitted.

Reply:
I think including explicitly that "redundant interfaces are used for increased fault tolerance, not for load sharing reasons.". can clear up misunderstandings and should be included in the text itself and not as a footnote. 

I don't think "kept in mind" is a very good phrasing since this hasn't been mentioned in the text before.